packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,221 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop         | Type                                                                     | Default     | Description                                                                                                                             | Required |
+| ------------ | ------------------------------------------------------------------------ | ----------- | --------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content      | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                  | ✅       |
```

Comment:
Is it possible to add styling to the text inside tooltip? For example, bold, italics?

Reply:
+1, we've had similar use-cases where content had to be styled. What's the challenge of having JSX instead of string?

Reply:
@cseas @akkhil7 discussed this with design. 

We will only allow plain strings for now. 

We will be standardising the design to not have any bold/italic text. We've done design assessment of the Tooltip content & blade advocates are in-lined with just plain text on the tooltip. 

> +1, we've had similar use-cases where content had to be styled.

@akkhil7 these existing cases will also get migrated (first from the design side) when adopting blade, 
Having said that if there could be genuine usecases for bold/italic texts we can accommodate that later.